tests/system/runtimes/test_nuclio.py

```diff
@@ -52,6 +53,29 @@ def test_deploy_function_with_error_handler(self):
         self._logger.debug("Deploying nuclio function")
         function.deploy()
 
+
+@tests.system.base.TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestNuclioRuntimeWithStream(tests.system.base.TestMLRunSystem):
+    project_name = "does-not-exist-4"
```

Comment:
you copy pasted from a test that purposed to test whether a project is being created automatically, here it's not the case, let's give a proper name `nuclio-stream-project` or something